test/gc/test-net-timeout.js

```diff
@@ -19,7 +19,7 @@ function serverHandler(sock) {
 
 const net = require('net');
 const weak = require('weak');
-require('../common');
+const common = require('../common');
```

Comment:
If you're touching this line, it might be worthwhile to move it to the top of the file.